src/biome/text/configuration.py

```diff
@@ -58,10 +58,10 @@ def from_params(
         cls: Type["FeaturesConfiguration"], params: Params, **extras
     ) -> "FeaturesConfiguration":
 
-        word = params.pop("word", params.pop("words", None))  # TODO: remove backward
+        word = params.pop("word", params.pop("words", None))
```

Comment:
I would prefer to remove the possibility of providing a `words` feature (if it does not break anything we already shipped). Otherwise i am pretty sure that at some point i will look up if the namespace of the vocabulary is `word` or `words`.

Reply:
+1 I would remove `words` too